### ISSUE TITLE
Encryption does not work for some specific value

### ISSUE THREAD

**Issue by RippleLeaf** [tonyo/pyope#13]
```python
from pyope.ope import OPE, ValueRange

cipher = OPE(b'fresh key', in_range=ValueRange(0, 1000000),
                              out_range=ValueRange(0, 2**31 - 1))

print(cipher.encrypt(926546)) # work
print(cipher.encrypt(926542)) # work
print(cipher.encrypt(926543)) # does not work
print(cipher.encrypt(926544)) # does not work
print(cipher.encrypt(926545)) # does not work
```

The last 3 lines fail at `assert in_size <= out_size`.
I think something in the recursive call might be wrong.

Thanks!

**Comment by hwfhc** [comment]
we encountered this problem too

**Comment by tonyo** [comment]
@RippleLeaf @hwfhc  Thanks for the report, I'll try to check what is wrong next week.

**Comment by tonyo** [comment]
Should be fixed in version 0.2.0.
